projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_grid.scss

```diff
@@ -197,10 +205,19 @@ $_light-grid: extend(
         row-hover-text-color: (
             igx-contrast-color: ('grays', 200)
         ),
+
         row-border-color: (
             igx-color: ('grays', 300)
         ),
 
+        row-col-selected-cell-background: (
+            igx-color: ('secondary', 50)
+        ),
+
+        row-col-selected-cell-color: (
+            igx-contrast-color: ('secondary', 50)
+        ),
+
```

Comment:
This always gets overridden by the code executed on lines 258 through 264 in `_grid-theme.scss`;